assets/pages/oneoff-contributions/components/contributionsCheckoutContainer.jsx

```diff
@@ -9,23 +9,16 @@ import ContributionsCheckout from 'components/contributionsCheckout/contribution
 
 import { type PageState as State } from '../oneOffContributionsReducer';
 import OneoffContributionsPayment from './oneoffContributionsPayment';
-import OneoffInlineContributionsPayment from './oneoffInlineContributionsPayment';
 
 
 // ----- State Maps ----- //
 
 function mapStateToProps(state: State) {
-
-  const inlineCardPaymentVariant = state.common.abParticipations.inlineStripeFlowCardPayment;
-
   return {
     amount: state.page.oneoffContrib.amount,
     currencyId: state.common.internationalisation.currencyId,
     country: state.common.internationalisation.countryId,
-    inlineCardPaymentVariant,
-    payment: inlineCardPaymentVariant === 'inline' ?
-      <OneoffInlineContributionsPayment /> :
-      <OneoffContributionsPayment />,
+    payment: <OneoffContributionsPayment />,
```

Comment:
In response to your comment [here](https://github.com/guardian/support-frontend/pull/745#discussion_r198156850), this component could be passed through in `oneoffContributions.jsx`, which means we can drop the `import React from 'react';` line.

```jsx
<ContributionsCheckoutContainer
  contributionType="ONE_OFF"
  form={<FormFields />}
  payment={<OneoffContributionsPayment />}
/>
```

Reply:
Fixed here: a163628 good spot!